c7n/resources/ecs.py

```diff
@@ -605,10 +607,17 @@ class RemoveTagEcsResource(RemoveTag):
                     tags: ["BadTag"]
     """
     permissions = ('ecs:UntagResource',)
+    batch_size = 1
 
     def process_resource_set(self, client, resources, keys):
+        old_arns = 0
         for r in resources:
+            if not ecs_taggable(self.manager.resource_type, r):
+                old_arns += 1
+                continue
             client.untag_resource(resourceArn=r[self.id_key], tagKeys=keys)
+        if old_arns != 0:
+            self.log.info("Couldn't untag %d resource(s). Needs new ARN format" % old_arns)
```

Comment:
Would recommend using new style string formatting

Reply:
stdlib logging is not compatible with format strings, and is lazy in its evaluation of string rendering.